### ISSUE TITLE
"Please Select Class" Error in the Train Search Page

### ISSUE THREAD

**Issue by Nilanjan-B** [shivamguys/irctc-cypress-automation#76]
![image](https://github.com/user-attachments/assets/7443876a-8924-4776-a824-bde322f15ed7)

This error is happening for each and every train from today 18th October 2024. What is the error or bug? Please help me to get out of this.

**Comment by shivamguys** [comment]
I know something has been changed from irctc side. Will try to fix this weekend.... Many people are reporting the same...

**Comment by ramchandra06032004** [comment]
error is it clicking on "Book Now" button of 1st train in results after clicking on Travel date

**Comment by shivamguys** [comment]
You are right I have fixed that Bug can you please check if its working .. :-) 
Please use the latest code

<img width="1079" alt="image" src="https://github.com/user-attachments/assets/4edf8114-c09d-4631-a27c-b14b8880156d">

**Comment by ramchandra06032004** [comment]
Okay I'll see tomorrow
